src/views/Budget/Header.tsx

```diff
@@ -143,7 +143,7 @@ export default function BudgetHeader({ scrollRef, onClick }: Props) {
         </div>
       </div>
       <button
-        name={formatDateKey(new Date())}
+        name={formatDateKey(getToday())}
```

Comment:
👍 
good catch!
The helper exists so that we can set a custom date as "today" in tests. So when we're adding a test for the today button this will help!